CHANGELOG.md

```diff
@@ -1,26 +1,28 @@
 X.Y.Z Release notes (YYYY-MM-DD)
 =============================================================
+NOTE: This version has been pre-released as v3.7.0-alpha.0 and v3.7.0-alpha.2. We have bumped major version due to removing functionality. Unfortunately, we also have an series of pre-releases of v4.0.0 which are incompatible with this version as the Realm file format is changed. We have decided to skip version v4.0.0, and the next major version (expected to be v6.0.0) will change file format.
```

Comment:
```suggestion
NOTE: This version has been pre-released as v3.7.0-alpha.0 and v3.7.0-alpha.2. We have bumped the major version due to removed functionality in this release. We are releasing this as 5.0.0 instead of 4.0.0 as we also have an series of pre-releases of v4.0.0 which are incompatible with this version as the Realm file format is upgraded. We have therefore decided to skip version v4.0.0, make this version 5.0.0 and we will then release the 4.0.0 version as the next major version - v6.0.0.
```

Reply:
Not sure if above is better?